xen/arch/arm/tee/ffa.c

```diff
@@ -107,20 +110,9 @@ static bool ffa_get_version(uint32_t *vers)
     return true;
 }
 
-static int32_t ffa_features(uint32_t id)
+static int32_t ffa_feature_supported(uint32_t id)
```

Comment:
Shouldn't the return type be `bool` instead?

Reply:
Will fix